### PR TITLE
Fixed 64bit enum support for structures where size was calculated incorrectly

### PR DESCRIPTION
**Pull request by Lukas-Dresel**
Fix for #5121

### CODE REVIEW

**Comment by radare** [comment]
do not merge in branches

File: libr/util/p_format.c [hunk 1]
```diff
@@ -1018,7 +1018,10 @@ int r_print_format_struct_size(const char *f, RPrint *p, int mode) {
 				case 1: size+=1; break;
 				case 2: size+=2; break;
 				case 4: size+=4; break;
-				default: break;
+				case 8: size+=8; break;
+				default:
+					eprintf("Unknown enum format size: %d\n", tabsize);
```
Comment (radare) [inline libr/util/p_format.c]: missing space before (

File: libr/util/p_format.c [hunk 2]
```diff
@@ -1018,7 +1018,10 @@ int r_print_format_struct_size(const char *f, RPrint *p, int mode) {
 				case 1: size+=1; break;
 				case 2: size+=2; break;
 				case 4: size+=4; break;
-				default: break;
+				case 8: size+=8; break;
```
Comment (radare) [inline libr/util/p_format.c]: missing spaces around += (same for previous)

**Comment by Lukas-Dresel** [comment]
I added test-cases for the pfo structures in radare2-regressions that should cover this case

**Comment by radare** [comment]
do not merge in branches you are pullreqesting

**Pull request merged**